src/common/models/Projection.ts

```diff
@@ -426,13 +415,19 @@ export class Projection {
   ) {
     const timeseriesRaw = summaryWithTimeseries.timeseries;
     const actualsTimeseriesRaw = summaryWithTimeseries.actualsTimeseries;
+    const metricsTimeseriesRaw = summaryWithTimeseries.metricsTimeseries;
     assert(
       actualsTimeseriesRaw.length > 0,
       `FIPS ${this.fips} missing actuals timeseries!`,
     );
     let earliestDate, latestDate;
     // If we have projections, we use that time range; else we use the actuals.
-    if (timeseriesRaw.length > 0) {
+    // TODO(chris): Is there a reason that this was bound to the projections timeseries first?
+    // It cuts off some of the earlier dates
```

Comment:
It's helpful to have a consistent start date so that we know that graphs for different locations will still render the same (so we can make sure it'll be legible at different resolutions, etc.)  Originally we did that by pinning everybody to the projections start date.

But this code is a mess and I think we could maybe just hardcode a start date of 3/1 at this point.

FWIW- This aligning code has always been kinda' yucky (I've been thinking we should just have a proper `Timeseries` class that wraps a timeseries and is date-aware)...  and we're probably really close to not needing it anymore, since we're no longer doing calculations between different timeseries that may not be date-aligned, since that's now in the backend...

Reply:
I think this will actually give a much more consistent start date since most of the metrics timeseries code goes back to early februrary.

Reply:
Also agree with the idea of a timeseries class of sorts! This was a good thought exercise to see where that might be useful.  Maybe part of it is breaking up the `Projection` class to be less powerful

Reply:
Do we think that early February data is useful to show?

Reply:
We do not! but the code below will standardize it to starting at march 1st.  Maybe we could remove the earliest time all together and just find an appropriate latest time?

Reply:
Yeah, I think that would be fine.  Like I said, this code has evolved into weirdness and doesn't make a ton of sense anymore.